sky/task.py

```diff
@@ -1101,6 +1145,9 @@ def add_if_not_none(key, value, no_empty: bool = False):
 
         add_if_not_none('resources', tmp_resource_config)
 
+        if self.vpn_config:
```

Comment:
```suggestion
        if self.vpn_config is not None:
```
nit: always use `if xx is (not) None` to check None value. Please check other palces in the PR.
https://google.github.io/styleguide/pyguide.html#2144-decision